changelog/v0.27.2/expose-morechangelogstuff.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
+  - type: NON_USER_FACING
+    description: >
+      Expose a feature of changelogutils to allow for more control over the changelog outputs
```

Comment:
This is pretty vague, needs a more descriptive message.

Reply:
Updated.